pybaseball/datahelpers/caching.py

```diff
@@ -0,0 +1,313 @@
+import copy
```

Comment:
This file is too much.

Reply:
In what way?

Reply:
100 LOC target

Reply:
That seems like a very aggressive target for a subsystem like this one. I'll see what I can do about splitting it up, but as a hard limit that may be difficult.

Reply:
I'll try and take a knife to it

Reply:
I don't want to split it up, I want to put it on a diet.